src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -15,6 +15,18 @@
 MSI_LOCAL_ID = '[system]'
 
 
+def get_target_network_api(cli_ctx):
+    """ Since most compute calls don't need advanced network functionality, we can target a supported, but not
+        necessarily latest, network API version is order to avoid having to re-record every test that uses VM create
+        (which there are a lot) whenever NRP bumps their API version (which is often)!
```

Comment:
Since this is used in the real product, it's not really about tests.
I would suggest removing the comment about re-recording tests.

Reply:
I call out that it exists for the tests because otherwise someone might rightly say "there's no harm in using latest if my profile is latest" and remove in the name of simplifying code. Tests explain why this code snippet even exists, and hint at the potential impact if this is changed.